### PR TITLE
Create database script

### LINKED ISSUE #20: Make Database Schema
## Description
Create a Schema.sql to create the tables as described in the ERD diagram.

## Required Files
- schema.sql

## User Story
As an engineer I need data to be stored in an organised way so I can easily use, access and query the data.

### PR DESCRIPTION
**Pull request by annalisev**
Closes #20

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    tag_id INT NOT NULL,
+    PRIMARY KEY (subscription_id),
+    FOREIGN KEY(subscriber_id) REFERENCES subscriber(subscriber_id),
+    FOREIGN KEY(tag_id) REFERENCES tag(tag_id)
+);
+
+CREATE TABLE game (
+    game_id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(100) NOT NULL,
+    price FLOAT NOT NULL,
```
[the remark this answers is not in the document]
Comment (annalisev) [reply database/schema.sql:51]: Good point! Will need to check this

**Comment by annalisev** [comment]
This should fix our website problem.

**Review by hhm970 — APPROVED** [review]
Looks great!

**Review by Eibironke — APPROVED** [review]
👍

**Pull request merged**